test/test_examples.py

```diff
@@ -24,6 +23,7 @@
     'capture_recapture.py --num-samples 4 --num-warmup 1 -m 5',
     'covtype.py --algo HMC --num-samples 10 --num-warmup 10',
     'gp.py --num-samples 10 --num-warmup 10 --num-chains 2',
+    'hmcecs.py --subsample_size 5 --num_svi_steps 1 --num_blocks 1 --num_warmup 1 --num_samples 5 --num_datapoints 100',
```

Comment:
does that mean the huge dataset gets downloaded in CI? can you instead add a `dataset=['mock', 'higgs']` and create a simple fake dataset for testing on the fly?

Reply:
Yes, I did not think of that. Would you prefer adding a parameter to the example over mocking `_load_higgs` in the test?

Reply:
whatever you think is simplest and most transparsent?

Reply:
I forgot how cumbersome `patching` is in python; I'll stick to your suggestion.